test/functional/ui/decorations_spec.lua

```diff
@@ -767,16 +768,16 @@ describe('decorations providers', function()
     end
     ]]
 
-    screen:expect{grid=[[
-      {2:Error in decoration provider ns1.start:} |
```

Comment:
Replying to your deleted comment @justinmk. Yes maybe this is a reasonable field format actually, it looked like a typo to me in the source😅

Reply:
I kind of like it, but not sure if that pattern is understood by many. The `(ns=...)` is probably a more broadly useful pattern for now.